posthog/settings.py

```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
     # Add hard-coded feature flags for static self-hosted releases here
     "5440-multivariate-support",
     "new-paths-ui-edge-weights",
+    "sigma-analysis",
```

Comment:
CC @neilkakkar

Reply:
This is released to everyone on Cloud, so all good! I'd actually want to just get rid of the FF.